README.md

```diff
@@ -68,6 +69,9 @@ format>
 
 Where:
 
+--request-source <string>
+replay source description
```

Comment:
Can we expand this description? As it stands I don't really understand what it would be used for. We could also consider including an example.